### PR TITLE
update caper test kits (A-SO, M-RRH, S-ES)

### PR DESCRIPTION
**Pull request by dtgreiner**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

Include a summary of the changes and a link to the related issue. List any dependencies that are required for this change.

## Type of change
[//]: # 'remove options that are not relevant'
- [X] Code clean-up / housekeeping

## Checklist before requesting review
- [X] I have performed a self-review of my code
- [X] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [X] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [X] My code follows the style guidelines of this project (rubocop)
- [X] I have updated the documentation (or not applicable)
- [ ] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: drivers/hud_apr/app/models/hud_apr/generators/shared/fy2024/question_twenty_three.rb [hunk 1]
```diff
@@ -59,26 +59,26 @@ def q23e_destination_type_by_race_and_ethnicity
...
         # permanent
         sheet.update_cell_members(
           cell: [letter, 5],
-          members: group_scope.where(a_t[:destination].in([422, 423, 426, 410, 435, 421, 411])),
+          members: group_scope.where(a_t[:destination].in(400..499)),
         )
         sheet.update_cell_members(
           cell: [letter, 6],
-          members: group_scope.where(a_t[:destination].in([8, 9, 17, 24, 30, 99]).or(a_t[:destination].eq(nil))),
+          members: group_scope.where(a_t[:destination].in(HudUtility2024.other_destinations).or(a_t[:destination].eq(nil))),
```
Comment (eanders) [inline drivers/hud_apr/app/models/hud_apr/generators/shared/fy2024/question_twenty_three.rb]: Is `HudUtility2024.other_destinations` equivalent to `1..99`?  Maybe we should follow the pattern above?  (That said, if it's a random list of ids, then maybe we should delegate to the HUD utility.

Comment (dtgreiner) [reply drivers/hud_apr/app/models/hud_apr/generators/shared/fy2024/question_twenty_three.rb]: I've updated this to 1..99 in the most recent push

**Comment by dtgreiner** [comment]
@eanders the rails test and other destinations have been added to the PR

**Comment by eanders** [comment]
@dtgreiner I'm going to merge this since the tests are passing now, and they're broken on 117 because we moved the fixpoints etc.  Feel free to open a new PR with the household changes when that's ready.

**Pull request merged**